docs/layouts/shortcodes/generated/forst_configuration.html

```diff
@@ -0,0 +1,120 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.cache.dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The directory where ForSt caches its SST files, fallback to state.backend.forst.local-dir/cache if not configured.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.cache.reserve-size</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Long</td>
+            <td>The reserved size of cache, when set to a positive number, dynamic space checking will be leveraged. This option and the state.backend.forst.cache.size-based-limitoption can be set simultaneously, the smaller cache limit will be used as the upper limit.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.cache.size-based-limit</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Long</td>
+            <td>The size-based capacity limit of cache, a non-positive number indicates that there is no limit.</td>
```

Comment:
we mention -1 as defaults for others - why is this non- positive . We should be consistent